test/price_rule_test.rb

```diff
@@ -43,7 +43,7 @@ def test_create_price_rule
     )
 
     assert_equal '{"price_rule":{"target_type":"line_item","allocation_method":"across","value_type":"fixed_amount","value":-10.0,"customer_selection":"all","starts_at":"2017-01-19T00:00:00Z"}}', WebMock.last_request.body
-    assert_equal -10, price_rule.value
+    assert_equal(-10, price_rule.value)
```

Comment:
I don't understand why it's correcting this but not the other `assert_equal` calls?

Reply:
This one in particular complained about the `-10` as it's not clear whether it's a parameter or not. We have ~600 infractions on the check for calling methods without parameters, but that's a lower severity warning. We'll get there 😄